code/datums/wounds/robotic/robotic_burns.dm

```diff
@@ -0,0 +1,381 @@
+
+	cooling_threshold = (BODYTEMP_NORMAL + 100)
+	heating_threshold = (BODYTEMP_NORMAL + 500)
+
+	cooling_demote_buffer = 60
+	heating_promote_buffer = 100
+
+	a_or_from = "from"
+
+	// easy to get
+	threshold_minimum = 30
+	threshold_penalty = 30
+
+	status_effect_type = /datum/status_effect/wound/burn/robotic/moderate
+
+	sound_volume = 20
+
+	outgoing_bodytemp_coeff = 0.009
+	bodytemp_coeff = 0.02
+
+	base_reagent_temp_coefficient = 0.05
+	heat_shock_delta_to_damage_ratio = 0.2
+
+	promotes_to = /datum/wound/burn/robotic/overheat/severe
+
+	light_color = COLOR_RED
+	light_power = 0.1
+	light_range = 0.5
+
+	can_scar = FALSE
+
+/datum/wound_pregen_data/burnt_metal/transient_overheat
+	abstract = FALSE
+
+	wound_path_to_generate = /datum/wound/burn/robotic/overheat/moderate
+
+/datum/wound/burn/robotic/overheat/severe
+	name = "Thermal Overload"
+	desc = "Exterior plating has surpassed critical thermal levels, causing significant failure in structural integrity and overheating of internal systems."
+	occur_text = "sizzles, the externals turning a dull shade of orange"
+	examine_desc = "appears discolored and polychromatic, parts of it glowing a dull orange"
+	treat_text = "Isolation from physical hazards, and accomodation of passive heat dissipation - active cooling may be used, but temperature differentials significantly \
+				raise the risk of thermal shock."
+	severity = WOUND_SEVERITY_SEVERE
+
+	a_or_from = "from"
+
+	threshold_minimum = 80
+	threshold_penalty = 65
+
+	status_effect_type = /datum/status_effect/wound/burn/robotic/severe
+
+	damage_mulitplier_penalty = 1.3 // 1.3x damage taken
+
+	starting_temperature_min = (BODYTEMP_NORMAL + 550)
+	starting_temperature_max = (BODYTEMP_NORMAL + 600)
+
+	heating_promote_buffer = 150
+
+	cooling_threshold = (BODYTEMP_NORMAL + 375)
+	heating_threshold = (BODYTEMP_NORMAL + 800)
+
+	outgoing_bodytemp_coeff = 0.009
+	bodytemp_coeff = 0.017
+
+	base_reagent_temp_coefficient = 0.03
+	heat_shock_delta_to_damage_ratio = 0.2
+
+	demotes_to = /datum/wound/burn/robotic/overheat/moderate
+	promotes_to = /datum/wound/burn/robotic/overheat/critical
+
+	light_color = COLOR_BRIGHT_ORANGE
+	light_power = 0.8
+	light_range = 0.5
+
+	scar_keyword = "robotic_burnsevere"
+
+/datum/wound_pregen_data/burnt_metal/transient_overheat
+	abstract = FALSE
+
+	wound_path_to_generate = /datum/wound/burn/robotic/overheat/moderate
+
+/datum/wound/burn/robotic/overheat/critical
+	name = "Runaway Exothermy"
+	desc = "Carapace is beyond melting point, causing catastrophic structural integrity failure as well as massively heating up the subject."
+	occur_text = "turns a bright shade of radiant white as it sizzles and melts"
+	examine_desc = "is a blinding shade of white, almost melting from the heat"
+	treat_text = "Immediate confinement to cryogenics, as rapid overheating and physical vulnerability may occur. Active cooling is inadvised, \
+				since the thermal shock may be lethal with such a temperature differential."
```

Comment:
```suggestion
	treat_text = "Immediate confinement to cryogenics, as rapid overheating and physical vulnerability may occur. Active cooling is not advised, \
		since the thermal shock may be lethal with such a temperature differential."
```